erpnext/crm/report/lead_details/lead_details.py

```diff
@@ -69,6 +70,10 @@ def get_columns():
 
 
 def get_data(filters):
+	if filters.get("from_date") and filters.get("to_date"):
```

Comment:
@Komal-Saraf0609 This doesn't seem like a correct fix, can you please elaborate on how did you reach this fix?